File: GanttChartLightLibraryDemos/Demos/Samples/WPF-CSharp/GanttChartDataGrid/GridColumns/GanttChartItemAttachments.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Windows;

namespace Demos.WPF.CSharp.GanttChartDataGrid.GridColumns
{
    static class GanttChartItemAttachments
    {
        public static readonly DependencyProperty MyValue5Property =
            DependencyProperty.RegisterAttached("MyValue5", typeof(string), typeof(GanttChartItemAttachments));
        public static string GetMyValue5(DependencyObject obj)
        {
            return (string)obj.GetValue(MyValue5Property);
        }
        public static void SetMyValue5(DependencyObject obj, string value)
        {
            obj.SetValue(MyValue5Property, value);
        }

        public static readonly DependencyProperty MyValue6Property =
            DependencyProperty.RegisterAttached("MyValue6", typeof(string), typeof(GanttChartItemAttachments));
        public static string GetMyValue6(DependencyObject obj)
        {
            return (string)obj.GetValue(MyValue6Property);
        }
        public static void SetMyValue6(DependencyObject obj, string value)
        {
            obj.SetValue(MyValue6Property, value);
        }
    }
}

File: GanttChartLightLibraryDemos/Demos/Samples/WPF-VisualBasic/GanttChartDataGrid/GridColumns/GanttChartItemAttachments.vb
Imports System.Text

Friend NotInheritable Class GanttChartItemAttachments

    Private Sub New()
    End Sub

    Public Shared ReadOnly MyValue5Property As DependencyProperty = DependencyProperty.RegisterAttached("MyValue5", GetType(String), GetType(GanttChartItemAttachments))
    Public Shared Function GetMyValue5(obj As DependencyObject) As String
        Return CStr(obj.GetValue(MyValue5Property))
    End Function
    Public Shared Sub SetMyValue5(obj As DependencyObject, value As String)
        obj.SetValue(MyValue5Property, value)
    End Sub

    Public Shared ReadOnly MyValue6Property As DependencyProperty = DependencyProperty.RegisterAttached("MyValue6", GetType(String), GetType(GanttChartItemAttachments))
    Public Shared Function GetMyValue6(obj As DependencyObject) As String
        Return CStr(obj.GetValue(MyValue6Property))
    End Function
    Public Shared Sub SetMyValue6(obj As DependencyObject, value As String)
        obj.SetValue(MyValue6Property, value)
    End Sub
End Class
